src/ZfBugsnag/Service/BugsnagService.php

```diff
@@ -67,33 +67,49 @@ public function setupErrorHandlers()
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0, $errcontext = [])
     {
-        $result = $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        try
+        {
+            $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        } catch (\Exception $e) {
+            // Something wrong in the bugsnag notify call
```

Comment:
What about using the ZendLogLogger/ZendLogWriterStream ?

https://akrabat.com/simple-logging-of-zf2-exceptions/